examples/03_solvers.jl

```diff
@@ -22,7 +26,7 @@ function ode_solver(
   prob = ODEProblem(f, u₀, time_span)
   sol = solve(prob, solver_alg; kwargs...)
   uₜ = sol.u[end]
-  return ITensor_from_vec(uₜ), nothing
+  return ITensor_from_vec(uₜ), ODEInfo(-1)
```

Comment:
What's the point of always returning `-1` here? Can we just leave this how it was?

Reply:
The reason I did this is to maintain some kind of return type stability between the different solvers. I suppose what you're getting at is can't we just check for 'nothing' in tdvp_site_update where the solvers are called, and handle it there. As the number of solvers will grow, having a uniform interface between all the solvers will make it much easier to reason about what data is available after you call them. The downside is that you have to keep the interface consistent, but I wrote the code so that the tests will fail if you forget, according to the fail quickly, fail early principle.

Reply:
I just mean that the current way it is written, it isn't really giving any information, so we can just keep it as is for now and return some convergence information when we have something useful to return.

Reply:
Returning `ODEInfo(-1)` vs `nothing` won't make a difference for type stability, and this type stability wouldn't matter much anyway.

Reply:
I have reverted this code change, as you requested